code/game/atoms_movable.dm

```diff
@@ -288,6 +288,31 @@
 		return 1
 	return 0
 
+/atom/movable/Cross(atom/movable/mover, turf/target, var/ignoreborder = 1)
+	if(flags & ON_BORDER && ignoreborder)
+		return 1
+	if(CrossCheck(mover, target))
+		return 1
+	if(!density || loc.contents[1] == src || !isturf(loc))
+		return 0
+	if(ismob(src))
+		to_chat(world, "[src]")
```

Comment:
Obviously I'm going to before it's otherwise actually ready to merge

Actually I already did but I haven't pushed that commit yet